content/en/Platform Deep Dive/Findings/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Findings"
+linkTitle: "Findings"
+weight: 15
+no_list: true
+description: >
+  Review and remediate findings that our pentesters discover.
+---
+
+A finding represents a potential security vulnerability or weakness identified during a security assessment. These assessments can encompass various methodologies, including:
+- **Penetration Tests**: In-depth manual assessments where security experts simulate real-world attacks to uncover vulnerabilities.
+- **Secure Code Reviews**: Manual analysis of source code to detect security vulnerabilities early in the software development lifecycle.
+- **Digital Risk Assessments**: Evaluations of your online presence to identify potential threats to your brand reputation, data security, and compliance.
+- **DAST Scans**: Automated scans that analyze running web applications and APIs to identify security flaws.
+- **Attack Surface Scans**: Insights derived from continuous attack surface monitoring, which automatically discovers and tracks your internet-facing assets for potential risks.
+
+Regardless of the test type, findings detail the identified vulnerability, its potential impact, and recommendations for remediation. This information empowers you to prioritize and address security gaps effectively.
+
+{{< alert title="Note" color="primary" >}}
+You can manage findings using the Cobalt API. Learn more in our [API documentation](/cobalt-api/documentation/v2/#findings).
+{{< /alert >}}
+
+## Organization View
+
+To view all findings reported within your organization, navigate to the **Findings** page. You need an [Organization Owner](/platform-deep-dive/collaboration/user-roles/#organization-owner) or [Member](/platform-deep-dive/collaboration/user-roles/#organization-member) role to view this page.
+
+{{% alert title="Note" color="primary" %}}
+Organization Members will be limited in their Findings view, depending on [group](/platform-deep-dive/collaboration/groups) membership.
```

Comment:
For some reason, this sentence is broken into two lines

<img width="666" alt="Screenshot 2024-12-09 at 15 53 39" src="https://github.com/user-attachments/assets/ba70ddf8-99db-45c1-97ef-4bdbb2167a20">

Reply:
I fixed this by updating the code from:
{{% alert title="Note" color="primary" %}}
{{% /alert %}}
to
{{< alert title="Note" color="primary" >}}
{{< /alert >}}